src/vcpkg/sourceparagraph.cpp

```diff
@@ -65,11 +65,11 @@ namespace
                                  "There was a close parenthesis without an opening parenthesis.");
     DECLARE_AND_REGISTER_MESSAGE(LicenseExpressionExpectLicenseFoundCompound,
                                  (msg::value),
-                                 "",
+                                 "Example of {value} is 'AND'",
```

Comment:
```suggestion
                                 "Example of {value} is 'AND'. The value is a boolean keyword and is intentionally not translated.",
```

or similar, ditto elsewhere.

Reply:
My opinion here is that these `{blah}`s are placeholders; we don't need to comment them up with "these won't be localized". The localization team doesn't need to care about what gets placed in there.

Reply:
I could see the localization team seeing "and" and assuming there will be a conjunction in that position if the target language has special rules for conjunctions.